### PR TITLE
feature/7: feedback from hackathon addressed

### LINKED ISSUE #7: Feedback from hackathon
Overall, it went well. A couple pointers I picked up from talking to people:

1. The placeholders in the curl examples were hard to make out as `:data_set_id`. I think we need to change the urls to be like `.../data_sets/{DATA_SET_ID}/version/{VERSION_IDENTIFIER}` and so on
2. Versions don't seem to be emphasized enough. It would be great to get some content in to explain why they are important w.r.t. how the data changes and is plugged in to
3. People didn't seem to find the section on access API keys or the API info for a data set easily. I think that this should be separated from the "how to use the API" section. I don't know if this is coming up in the how-to docs or if it was because people were skimming the current docs -- it's just feedback that I heard

### PR DESCRIPTION
**Pull request by tlysecki**
closes #7

### CODE REVIEW

File: source/includes/basics.md [hunk 1]
```diff
@@ -6,15 +6,21 @@ If you're looking to interact with the Namara API, you've come to the right plac
...
-<code>https://:namara_api_host/v0/:endpoint</code>
+<code>https://{NAMARA_API_HOST}/v0/{ENDPOINT}</code>
 
 Unless indicated otherwise, the Namara API host is `api.namara.io`, which will be used throughout the documentation. 
 
 The responses will be outlined for each endpoint.
 
+## Data Set Versions
+
+Our data is updated regularly and automatically, so values in rows and columns may change. The data set version is a snapshot of the data set's structure and properties. When that structure changes, we create a new version of the data set.
```
Comment (broneks) [inline source/includes/basics.md]: no comma in first sentence and make it "so _that_ values...". I don't think the last sentence needs a comma either.

File: source/includes/basics.md [hunk 2]
```diff
@@ -82,13 +64,35 @@ Here's how to verify your API Key:
...
 200: OK | The API Key is valid (*example 1*)
-422: Unprocessable Entity | API does not correspond to any user (*example 2*)
\ No newline at end of file
+422: Unprocessable Entity | API does not correspond to any user (*example 2*)
+
+### Rate Limiting
+
+Users are limited to 10,000 requests per month, as well as 100 data set downloads per month. When you exceed this limit, the API will return `status: 429`. If you find yourself meeting the limits, contact your account manager or <a href="mailto:info@thinkdataworks.com" target="_blank" rel="noreferrer noopener">email us</a> to find a solution.
+
+# API Keys
```
Comment (broneks) [inline source/includes/basics.md]: Should this section on API Keys be higher in the docs? I mean everything in the docs relies on the user knowing / having access to their key. Thoughts?

File: source/includes/basics.md (hunk 1 above)
Comment (tlysecki) [reply source/includes/basics.md]: That changes the meaning of the sentence to something other than what it should be. The grammar here is fine.

File: source/includes/basics.md (hunk 2 above)
Comment (tlysecki) [reply source/includes/basics.md]: This was pulled out of the "intro" section specifically to give it more visibility in the sidebar. It was higher in that intro section, and Chris gave me feedback from hackathoners who said that info was hard to find.

**Pull request merged**